src/importexport/guitarpro/internal/gtp/gpconverter.cpp

```diff
@@ -937,6 +937,20 @@ void GPConverter::addContiniousSlideHammerOn()
                 legatoSlides.erase(startNote);
                 legatoSlides[endNote] = slur;
             }
+
+            // TODO-gp: implement for editing too. Now works just for import.
+            if (slide.second == SlideHummerOn::HammerOn) {
```

Comment:
`SlideH**u**mmerOn::H**a**mmerOn`?

Reply:
haha right! once you read the same way, does it really matter? =)

Reply:
Coukld you please get that "Hummer" changed to "Hammer" all the way?